ortho_seq_code/orthogonal_polynomial.py

```diff
@@ -10,16 +10,17 @@
 import click
 import itertools
 from matplotlib import pyplot as plt
+from ortho_seq_code.plotclass import rf1d
 
 
 def create_dir_if_not_exists(out_dir):
     if os.path.exists(out_dir):
         ct = 0
         while os.path.exists(out_dir):
             if ct != 0:
-                loc = out_dir.rfind("[") - len(out_dir)
+                loc = -1 * len(str(ct - 1))
```

Comment:
I might be missing something around what this is doing. It looks to me like it's trying to calculate how much has been added to out_dir and remove it later to then add the new number. It seems like this would be simpler (if I'm not mistaken on what's happening here) if it was something like the following:

```suggestion
ct = 0
new_out_dir = f"{out_dir}0"
while os.path.exists(new_out_dir):
  ct += 1
  new_out_dir = f"{out_dir}{ct}
  
out_dir = new_out_dir
```
or it could just use the `new_out_dir` variable if reassignment isn't necessary.

Reply:
I think you missed some closing quotes in line 25 (I'm not familiar with this form of string manipulation), but otherwise it looks good! I will use this